MapboxMobileEvents/MMELocationManager.m

```diff
@@ -30,6 +33,7 @@ - (instancetype)init {
         _application = [[MMEUIApplicationWrapper alloc] init];
         NSArray *backgroundModes = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"UIBackgroundModes"];
         _hostAppHasBackgroundCapability = [backgroundModes containsObject:@"location"];
+        _configuration = [MMEEventsService.sharedService configuration];
```

Comment:
I missed this one `[MMEEventsService sharedService]` please make sure all calls to the `sharedService` follows the convention.

Reply:
Right, sorry about that. Thanks for catching those.